fdbserver/ClusterController.actor.cpp

```diff
@@ -1062,9 +1062,15 @@ class ClusterControllerData {
 				std::swap(regions[0], regions[1]);
 			}
 
-			if (regions[1].dcId == clusterControllerDcId.get() && regions[1].priority >= 0 &&
+			if (regions[1].dcId == clusterControllerDcId.get() &&
 			    (!versionDifferenceUpdated || datacenterVersionDifference >= SERVER_KNOBS->MAX_VERSION_DIFFERENCE)) {
-				std::swap(regions[0], regions[1]);
+				if (regions[1].priority >= 0) {
+					std::swap(regions[0], regions[1]);
+				} else {
+					TraceEvent(SevWarn, "DcPriorityNegative")
```

Comment:
Maybe change the severity to `SevWarnAlways`.

Also consider changing the event type name to `CCDcPriorityNegative`.

Reply:
Thanks, done.